platform/linux-generic/include/odp_schedule_if.h

```diff
@@ -24,14 +26,14 @@ typedef struct {
 	odp_queue_t (*queue_lookup)(const char *name);
 	int (*queue_capability)(odp_queue_capability_t *capa);
 	int (*queue_context_set)(odp_queue_t queue, void *context,
-				     uint32_t len);
+				 uint32_t len);
```

Comment:
@nagarahalli No. We have to look on all commits. Each commit should be clean and complete, not the 'all commits'.